src/Sulu/Component/Content/Tests/Unit/Repository/ContentRepositoryTest.php

```diff
@@ -164,8 +169,8 @@ public function setUp(): void
         $this->structureManager->getStructures('page')->willReturn([$structure->reveal()]);
         $this->structureManager->getStructure('test')->willReturn($structure->reveal());
 
-        $this->query = $this->prophesize(QueryInterface::class);
-        $this->query->setLimit(Argument::any())->willReturn(null);
+        $this->query = $this->prophesize(QueryObjectModel::class);
+        $this->query->setLimit(Argument::any());
```

Comment:
Looks like this change does make prefeer lowest failing.

![Bildschirmfoto 2024-04-04 um 16 22 54](https://github.com/sulu/sulu/assets/1698337/c1bb4f6b-cc97-4d9e-b4a9-8f3a1b221a3e)

@mamazu do you want to have a look at this?

Reply:
Looks like something here https://github.com/sulu/sulu/blob/9fb83560219c38e19f92fa6ad773f39490c81c8e/src/Sulu/Component/Content/Tests/Unit/Repository/ContentRepositoryTest.php#L182 is missing.

Reply:
Yes, PR is open.